app/main.go

```diff
@@ -36,8 +37,12 @@ func main() {
 	if err != nil {
 		log.Fatalf("[ERROR] can't open db %s, %v", opts.DB, err)
 	}
+	tg, err := proc.NewTelegramClient(opts.TG)
+	if err != nil {
+		log.Fatalf("[ERROR] failed initilization telegram client %s, %v", opts.TG, err)
+	}
```

Comment:
when starting without `TELEGRAM_TOKEN` there will be an error, 
- is this the right behavior?
- need to register this env in docker-compose?

Reply:
this is not right, telegram support should be optional

Reply:
for "need to register this env in docker-compose?" - I think you are asking if it should be passed via docker-compose? Yes, if user needs to set (or pass) `TELEGRAM_TOKEN` it has to be defined/declared in the compose

Reply:
Fixed